.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,8 +23,15 @@ Please also:
 - [ ] This PR meets the Acceptance Criteria in the JIRA story
 - [ ] The JIRA story has been moved to Dev Review
 - [ ] This PR includes appropriate unit tests
-- [ ] I have run and tested this change locally
-- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/main/e2e/README.md#examples)
-- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
-- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
+- [ ] I have added explanatory comments where the logic is not obvious
+- [ ] I have run and tested this change locally, and my testing process is described here
+- [ ] If this includes a UI change, I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
```

Comment:
If you have a **UI change** only as this bullet indicates, the presubmit will fully cover this, it's the dev's prerogative whether they want to run the tests locally as well.

If you have an API change that might impact the e2e tests, it would be responsible to run all or at least the relevant tests e2e locally, since it's not covered by presubmit.

Reply:
very good point, thanks